### ISSUE TITLE
Module not found: Can't resolve 'rosie' 

### ISSUE THREAD

**Issue by brandongregoryscott** [AndcultureCode/AndcultureCode.JavaScript.React#7]
When I attempted to switch over my project to use the published package version of this repo (I was previously referencing my local copy of the repo for development), the frontend no longer compiles due to a missing dependency `rosie`. This might be due to the tests being included in the final package bundle and `rosie` being a dev dependency. 

```
Failed to compile.

./node_modules/andculturecode-javascript-core/dist/tests/factories/axios-response-factory.js
Module not found: Can't resolve 'rosie' in '/Users/Brandon/SpotifySync/frontend/node_modules/andculturecode-javascript-core/dist/tests/factories'
```

**Comment by brandongregoryscott** [comment]
@wintondeshong I was able to resolve this by removing those export forwards from `src/tests/factories/index.ts`, but I'm wondering if we should be including those factories in the bundle at all. I think we have `**/*.test.*` as an exclude pattern in our tsconfig, but maybe the whole `tests/` folder should be excluded too.

**Comment by wintondeshong** [comment]
@brandongregoryscott Yeah, the goal was to eventually here loop around and extract it to the testing package. I'm just now changing gears to push through frontend actors more so I'll start by creating the AndcultureCode.JavaScript.Testing package which should resolve this issue. Thank you!

**Comment by wintondeshong** [comment]
@brandongregoryscott This should be good to go. Give this change a whirl and let me know if this resolves the issue. If it doesn't, we can re-open.

**Comment by wintondeshong** [comment]
This was published with v0.0.5

**Comment by wintondeshong** [comment]
While the fix was in v0.0.5, please test with v0.0.6
